ghcide/src/Development/IDE/GHC/Compat.hs

```diff
@@ -449,7 +449,7 @@ setUpTypedHoles df
   $ df
   { refLevelHoleFits = Just 1   -- becomes slow at higher levels
   , maxRefHoleFits   = Just 10  -- quantity does not impact speed
-  , maxValidHoleFits = Nothing  -- quantity does not impact speed
+  , maxValidHoleFits = Just 10  -- quantity does not impact speed
```

Comment:
outdated comment also on the previous line

Can we have a comment explaining the choice of the magic number? "I made this up" is fine, it indicates we don't have a good reason for it, which is helpful information for the future

Reply:
Is this the version we want? I don't really know.

Reply:
Fixed the comments.